lib/DXIL/DxilUtil.cpp

```diff
@@ -511,15 +532,7 @@ bool IsHLSLObjectType(llvm::Type *Ty) {
     if (IsHLSLResourceType(Ty))
       return true;
 
-    ConsumePrefix(name, "class.");
-    ConsumePrefix(name, "struct.");
-
-    if (name.startswith("TriangleStream<"))
-      return true;
-    if (name.startswith("PointStream<"))
-      return true;
-    if (name.startswith("LineStream<"))
-      return true;
+    return IsHLSLObjectStreamType(Ty);
```

Comment:
This change is no longer used, right?  So it seems almost unrelated to other things in the PR now, and not really necessary.